app/src/main/java/com/infomaniak/mail/data/cache/mailboxContent/CustomRefreshStrategies.kt

```diff
@@ -92,13 +91,14 @@ val snoozeRefreshStrategy = object : DefaultRefreshStrategy {
 
         MessageController.getMessage(remoteMessage.uid, realm)?.let { localMessage ->
             remoteMessage.initLocalValues(
-                messageInitialState = MessageInitialState(
-                    isFullyDownloaded = localMessage.isFullyDownloaded(),
-                    isTrashed = localMessage.isTrashed,
-                    isFromSearch = localMessage.isFromSearch,
-                    draftLocalUuid = localMessage.draftLocalUuid,
-                ),
+                isFullyDownloaded = localMessage.isFullyDownloaded(),
```

Comment:
Why can't we simply take the localMessage._isFullyDownloaded directly in this case? We don't need to compute anything right? We only need to take the existing localMessage as is, nothing more, nothing less

Reply:
Actually, we can.